Source/Pawnmorphs/Esoteria/Hediffs/Comp_MutationSeverityAdjust.cs

```diff
@@ -244,17 +240,17 @@ public virtual float SeverityChangePerDay()
             if (parent.Severity < minSeverity) sevPerDay = Mathf.Max(0, sevPerDay);
 
 
-            return sevPerDay * Mathf.Max(StatAdjust.Value, 0); //take the mutagen sensitivity stat into account 
+            return sevPerDay * Mathf.Max(StatsUtility.GetStat(Pawn, PMStatDefOf.MutagenSensitivity, 300) ?? 1, 0); //take the mutagen sensitivity stat into account 
         }
 
 
-        internal float EffectiveMax => Mathf.Max(MutationAdaptability.Value, 1) + SeverityOffset;
+        internal float EffectiveMax => Mathf.Max(StatsUtility.GetStat(Pawn, PMStatDefOf.MutationAdaptability, 300) ?? 0, 1) + SeverityOffset;
```

Comment:
Shouldn't the default for MutationAdaptability be 1?  Or are we assuming a pawn can't adapt at all unless they explicitly have the stat?  (I'd appreciate a comment specifying if it's the latter)

Reply:
I checked the stat definition and it's -2 to +2 with a default of 0. Apparently.